includes/admin/class-wc-rest-payments-settings-controller.php

```diff
@@ -550,8 +556,11 @@ private function update_is_wcpay_subscriptions_enabled( WP_REST_Request $request
 	 */
 	private function update_account( WP_REST_Request $request ) {
 		$updated_fields_callback = function ( $value, string $key ) {
-			return in_array( $key, static::ACCOUNT_FIELDS_TO_UPDATE, true ) &&
-					$this->wcpay_gateway->get_option( $key ) !== $value;
+			if ( ! ( in_array( $key, static::ACCOUNT_FIELDS_TO_UPDATE, true ) &&
+				$this->wcpay_gateway->get_option( $key ) !== $value ) ) {
+				return false;
+			}
+			return ! in_array( $key, static::ACCOUNT_FIELDS_CANNOT_BE_EMPTY, true ) || ! empty( $value );
```

Comment:
@dpaun1985: not sure why we need changes here, would you please share insights?

Reply:
> @dpaun1985: not sure why we need changes here, would you please share insights?

I wanted to add a double check of email and phone also on backend side. If the fields are empty, we don't send the empty values to Stripe. 
Another option could be also to throw an error if the fields are sent with empty values.

Reply:
I see. Can we skip the changes?

There is JS validation and settings will not be saved if JS is not working - so realistically we'll not get empty values saved. Do I understand the context in the right way?

Reply:
yes. I will revert the changes here.

Reply:
Great, thanks!